### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ChPraveen97**
This will update README.md by adding Day1 work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # JavaScriptLearning
 Contains practice codes I did while learning JavaScript.
+ 	Day 1: Learning JavaScript Fundamentals.
```
Comment (ShivaniPR) [inline README.md:3]: Please add some more files, only then will I have something to review.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
 # JavaScriptLearning
 Contains practice codes I did while learning JavaScript.
```
Comment (ShivaniPR) [inline README.md:2]: PR title is not apt, also there is a spelling mistake. Please make sure you don't repeat such things

**Review by ShivaniPR — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ChPraveen97) [reply README.md:3]: Ok, I was just testing pull request option. I will add some files for next pull request.

File: README.md (hunk 2 above)
Comment (ChPraveen97) [reply README.md:2]: I'll keep this in mind and I will not repeat this.

Comment (ShivaniPR) [reply README.md:2]: Please update the PR title

**Pull request merged**